blocks/init/src/Blocks/components/button/button.php

```diff
@@ -44,11 +41,15 @@
 
 $buttonClass = Components::classnames([
 	Components::selector($componentClass, $componentClass),
+	Components::selector($blockClass, $blockClass),
 	Components::selector($blockClass, $blockClass, $selectorClass),
 	Components::selector($additionalClass, $additionalClass),
 	Components::selector($buttonIsAnchor, 'js-scroll-to-anchor'),
 ]);
 
+$unique = $attributes['uniqueWrapperId'] ?? Components::getUnique();
+echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Should we put this inside the `<button>` and `<a>` elements? it will add some small duplication, but will be scoped, and won't pollute the main DOM.

Reply:
there is no need to do this I will make changes to this output so that the location will not be relevant.